homework-submission/week1-react/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import todoLogo from './todo.png';
+import './App.css';
+import Todo from "./todo.js";
+import data from './todoData.json';
+
+class App extends Component {
+
+  state = {
+    title: 'My Todo List',
+    data: data
+  };
+
+  changeDone = (index) => {
+    const newState = this.state;
+    if(this.state.data[index].done) {
+      newState.data[index].done = false  
+    } else {
+      newState.data[index].done = true
+    }
+    this.setState(newState);
+    console.log(newState.data[index])
+  }
+ 
+  render() {
+    const elements = (this.state.data.length < 1 || this.state.data === undefined) ? <p>No items...</p> : this.state.data.map((item, key)=> <Todo
```

Comment:
what about the `!` operator?
So it can be as the following:
```javascript
    const isEmpty = !this.state.data.length || !this.state.data
    const elements = (isEmpty) ?(
      <p>No items...</p> : 
      this.state.data.map((item, key)=> <Todo
        //... etc
      />
      )
    )
```

Isn't better looking 😃 ?

As usual change as it fits, and expect me to ask you about what is inside. 🙏

Reply:
you are absolutely right, it was so confusing and now it is simple to read :) thanks